app/apps/planner/admin.py

```diff
@@ -23,6 +23,8 @@ class DaySettingsInline(admin.TabularInline):
                     "week_segments",
                     "priorities",
                     "reasons",
+                    "postal_code_ranges",
+                    "districts",
```

Comment:
Dit PR gaat allemaal over districts, waar komt postal_code_ranges vandaan

Reply:
district vervangt postal_code_ranges_presets. Maar de code hierboven gaat over de admin, alleen maar om te kijken of waardes goed doorkomen.